templates/config-sokol-m.json

```
@@ -0,0 +1,272 @@
+                "format": "u16",
+                "address": 74,
+                "type": "value",
+                "group": "channels",
+                "readonly": true
+            },
+            {
+                "name": "NO2",
+                "reg_type": "holding",
+                "format": "u16",
+                "address": 75,
+                "type": "value",
+                "group": "channels",
+                "readonly": true
+            },
+            {
+                "name": "SO2",
+                "reg_type": "holding",
+                "format": "u16",
+                "address": 76,
+                "type": "value",
+                "group": "channels",
+                "readonly": true
+            },
+            {
+                "name": "H2S",
+                "reg_type": "holding",
+                "format": "u16",
+                "address": 77,
+                "type": "value",
+                "group": "channels",
+                "readonly": true
+            },
+            {
+                "name": "HCN",
+                "reg_type": "holding",
+                "format": "u16",
+                "address": 78,
+                "type": "value",
+                "group": "channels",
+                "readonly": true
+            },
+            {
+                "name": "NH3",
+                "reg_type": "holding",
+                "format": "u16",
+                "address": 79,
+                "type": "value",
+                "group": "channels",
+                "readonly": true
+            },
+            {
+                "name": "CH2O",
+                "reg_type": "holding",
+                "format": "u16",
+                "address": 80,
+                "type": "value",
+                "group": "channels",
+                "readonly": true
+            }
+        ],
+        "parameters": [
+        ],
+        "translations": {
+            "en":  {
+                "Pm25":"Dust 2.5 microns",  
+                "Pm10": "Dust 10 microns",
+                "Pressure1": "Pressure",
+                "Pressure2": "Pressure"
+            },
+            "ru": {
+                "Temperature": "Температура воздуха",
+                "Pressure1": "Атмосферное давление",
+                "Pressure2": "Атмосферное давление",
+                "Relative humidity":"Относительная влажность воздуха",
+                "Wind speed": "Скорость ветра",
+                "Wind direction": "Направление ветра",
+                "Precipitation rate": "Уровень осадков",
+                "Illumination level": "Уровень освещенности",
+                "Wind speed ultrasonic": "Скорость ветра УЗ анемометра",
+                "Wind direction ultrasonic": "Направление ветра УЗ анемометра",
+                "Snow cover": "Снежный покров",
+                "Sun radiation": "Солнечная радиация",
+                "Pm25": "Пыль 2,5 мкм",
+                "Pm10": "Пыль 10 мкм",
+                "CO2": "Уровень CO2",
```

```suggestion
                "CO2": "Уровень CO₂",
```
Так же можно сделать для английского и для других надписей